src/utils/index.js

```diff
@@ -227,6 +230,9 @@ export const isRunningLocally = () => {
  * @returns {string} Sanitized pathname
  */
 export const sanitizedPathname = () => {
+  if (typeof window === 'undefined' || !window.location) {
```

Comment:
same as above, to resolve this window issue

<img width="895" alt="Screenshot 2025-06-27 at 15 08 30" src="https://github.com/user-attachments/assets/736386a3-0d3f-472a-aaf9-6bb0867a27ab" />